packages/kit/src/exports/vite/index.js

```diff
@@ -671,14 +673,13 @@ Tips:
 								assetFileNames: `${prefix}/assets/[name].[hash][extname]`,
 								hoistTransitiveImports: false,
 								sourcemapIgnoreList,
-								manualChunks: split ? undefined : () => 'bundle',
-								inlineDynamicImports: false
+								manualChunks: isRolldown || split ? undefined : () => 'bundle',
+								inlineDynamicImports: split && isRolldown
```

Comment:
what's the difference between `manualChunks: () => 'bundle'` and `inlineDynamicImports: true`? could you simplify by combining the two lines? or would that produce worse output in the rollup case?
```suggestion
								inlineDynamicImports: !split
```